lib/rudder/pipeline.ex

```diff
@@ -63,38 +66,61 @@ defmodule Rudder.Pipeline do
           case Rudder.BlockResultUploader.upload_block_result(block_result_metadata) do
             {:ok, cid, block_result_hash} ->
               :ok = Rudder.Journal.commit(bsp_key)
+              bsp_upload_success()
```

Comment:
shouldn't this be `brp_upload_success` in place of `bsp_upload_success` ?

Reply:
it meant for of "pipeline upload failure", but it's confusing. Changing it now.